api/src/main/java/openconsensus/trace/data/SpanData.java

```diff
@@ -76,6 +76,7 @@ public static SpanData create(
       @Nullable Status status,
       @Nullable Timestamp endTimestamp) {
     Utils.checkNotNull(messageEvents, "messageEvents");
+    kind = kind == null ? Kind.INTERNAL : kind;
```

Comment:
Any reason to accept nullable kind for SpanData?

Reply:
I think we made `SpanKind` as an optional field before. After this change it may make more sense to make it required. Updated and removed the nullable annotation.